### PR TITLE
Fix/Support type and status fields in app context

### PR DESCRIPTION
**Pull request by seba-aln**
fix: Add missing status and type fields in app context. Now they are included, by default, in the response for getting channel/uuid metadata

### CODE REVIEW

File: pubnub/endpoints/objects_v2/channel/get_all_channels.py [hunk 1]
```diff
@@ -1,17 +1,18 @@
 from pubnub.endpoints.objects_v2.objects_endpoint import ObjectsEndpoint, ListEndpoint, \
-    IncludeCustomEndpoint
+    IncludeCustomEndpoint, IncludeStatusTypeEndpoint
```
Comment (jguz-pubnub) [inline pubnub/endpoints/objects_v2/channel/get_all_channels.py:2]: Is `IncludeStatusTypeEndpoint` interface common for include type and status?

Comment (seba-aln) [reply pubnub/endpoints/objects_v2/channel/get_all_channels.py:2]: Yeah. I've packed them into one class but you can manage them separately. I think when they always appear together there's no sense in separating them.

File: pubnub/endpoints/objects_v2/objects_endpoint.py [hunk 2]
```diff
@@ -100,8 +106,32 @@ def __init__(self):
 
     def custom(self, custom):
         self._custom = dict(custom)
+        self._include_custom = True
+        return self
+
+
+class StatusTypeAwareEndpoint:
+    __metaclass__ = ABCMeta
```
Comment (jguz-pubnub) [inline pubnub/endpoints/objects_v2/objects_endpoint.py:114]: What is `ABCMeta` here?

**Comment by seba-aln** [comment]
@pubnub-release-bot release

**Comment by seba-aln** [comment]
@pubnub-release-bot release

**Pull request merged**